doc/api/dgram.md

```diff
@@ -299,6 +299,84 @@ added: v0.6.9
 Sets or clears the `SO_BROADCAST` socket option.  When set to `true`, UDP
 packets may be sent to a local interface's broadcast address.
 
+### socket.setMulticastInterface(multicastInterface)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `multicastInterface` {String}
+
+*Note- All references to scope in this section are refering to
```

Comment:
This is fine for now, I think.. I'll be going through all the docs here soon to standardize the formats of the various `Note:` instances